lib/ansible_tower_client/collection.rb

```diff
@@ -11,37 +11,47 @@ def all
     end
```

Comment:
@bdunne - do we need to add a `.map {|x| x}` (or `.collect..`) to the `all` method?  I'm just getting an Enumerator class back.